model/src/main/resources/reference.yml

```diff
@@ -210,8 +210,6 @@ ort:
         # Command line options that do not affect the ScanCode output.
         commandLineNonConfig: '--processes 4'
 
```

Comment:
commit-msg: Would it make sense to explicitly say that ORT doesn't want to support any version below 3?

Reply:
At least it was not my intention to say that. But as soon as version 3 is *also* supported, the code becomes a bit cumbersome to maintain if the `parseLicenseExpressions` feature stays.